src/model/data.types.model.ts

```diff
@@ -16,3 +16,16 @@ export enum BeneficialOwnerTypeChoice {
     otherLegal = "otherLegal",
     none = "none"
 }
+
+
+export interface InputDate {
```

Comment:
Could we not just name this 'Date', or is there a name clash with a reserved word (or something)? All of these things are 'input'....

Reply:
"Date" clashes
We want the name to be as generic as possible.